server/android/makefile

```diff
@@ -31,4 +31,4 @@ android: $(ARCHS_ANDROID)
 
 .PHONY: $(ARCHS_ANDROID)
 $(ARCHS_ANDROID): %:
-	$(CARGO_LINKER_ENV_$@)="$(CC_$@)" CC="$(CC_$@)" AR="$(AR_$@)" cargo build --target $@ --release
+	$(CARGO_LINKER_ENV_$@)="$(CC_$@)" CC="$(CC_$@)" AR="$(AR_$@)" cargo build --features android --target $@ --release
```

Comment:
interesting - I'm still getting this when running `make android`

![Screen Shot 2022-06-08 at 9 22 58 AM](https://user-images.githubusercontent.com/9192912/172485649-d08d655c-6c33-44ee-969a-fca46b16d4b8.png)